setup.py

```diff
@@ -29,7 +29,7 @@
     },
     install_requires=[
         "googleapis-common-protos>=1.57",
-        "flyteidl>=1.5.6",
+        "flyteidl==1.5.9",
```

Comment:
wait til tests pass
```suggestion
        "flyteidl>=1.5.9,<1.6.0",
```

Reply:
it doesn't work, but `pip install "flyteidl>=1.5.8,<1.6.0"` works.

Reply:
maybe merge this [pr](https://github.com/flyteorg/flyteidl/pull/411), then update it to "flyteidl>=1.5.9,<1.6.1",